lib/Common/ConfigFlagsList.h

```diff
@@ -1492,6 +1492,9 @@ FLAGNR(Boolean, CFG, "Force enable CFG on jshost. version in the jshost's manife
 
 FLAGR(Number, JITServerIdleTimeout, "Idle timeout in seconds to do the cleanup in JIT server", 10)
 FLAGR(Number, JITServerMaxInactivePageAllocatorCount, "Max inactive page allocators to keep before schedule a cleanup", 10)
+
+FLAGNR(Boolean, StrictWriteBarrierCheck, "Check write barrier setting on none write barrier pages", false)
```

Comment:
Do we want this flag available on "Test" build? If yes, change caller RecyclerWriteBarrierManager to use it in "Test" build (DBG -> ENABLE_DEBUG_CONFIG_OPTIONS).